Source/WebKit/UIProcess/Automation/Automation.json

```diff
@@ -2,6 +2,59 @@
     "domain": "Automation",
     "description": "Automation domain exposes commands for automating user interactions with the browser.",
     "types": [
+        {
+            "id": "AccessibleNode",
+            "type": "object",
+            "description": "Snapshot of a DOM node's accessibility properties.",
+            "properties": [
+                { "name": "activeDescendantElement", "type": "string", "optional": true },
+                { "name": "atomic", "type": "boolean", "optional": true },
+                { "name": "autocomplete", "type": "string", "optional": true },
+                { "name": "busy", "type": "boolean", "optional": true },
+                { "name": "checked", "type": "string", "optional": true },
+                { "name": "colcount", "type": "integer", "optional": true },
```

Comment:
This may be following a spec, but the capitalization doesn't seem consistent. Here we have colcount which is all lowercase to match aria-colcount - but above we have activeDescendantElement when the aria property is aria-activedescendant